procurement.md

```diff
@@ -0,0 +1,695 @@
+remediation might still be easy and relatively inexpensive.
+
+Finally, small, modular contracts align well with a technologically
+modular design.  Modular contracting enforces modular technical design
+as different teams need to separate concerns so they can all develop
+in parallel.  Similarly, a technically modular product is susceptible
+to development by a variety of smaller vendors in ways that a
+monolithic structure would prevent.  While one *could* hire one vendor
+to build an entire system, teams that work together under one roof are
+more likely to --- indeed, almost inevitably will --- violate modular
+boundaries under deadline pressure.  Separating those teams helps
+enforce the technical boundaries that keep open source process
+functioning. Another approach to enforcing modular separation is
+through OSQA, described below.
+
+KEY RECOMMENDATION: Modular contracting works best with agile,
+  open source development and modular technical design.
+
+One aspect of modular contracting that is sometimes overlooked is that
+it can be costly to conduct many smaller rounds of procurement.  Even
+if those smaller rounds qualify for faster processes with less
+overhead, the logistics around each agreement are themselves a factor
+that introduce both delay and risk to a project.  It is quite common
+for all parties to agree on a set of work to be done and yet never
+proceed all the way through the contracting stage for various reasons.
+
+Vendors too see a rise in costs.  Because municipal scopes and budgets
+are often smaller than for federal projects, vendors at this level of
+government often find the small sizes of modular contracts a
+challenge.  Smaller contracts for shorter periods of work provide less
+stability.  Vendors find themselves unable to make the long-term
+commitments needed to hire long-term employees instead of ad-hoc
+contractors tend not to stay as long.  That raises their costs, which
+of course eventually raises costs for your agency.
+
+For these reasons, we suggest that agencies engaged in modular
+contracting place those contract modules in larger Master Services
+Agreements ("MSAs") while also encouraging vendors to seek
+multiple contract modules under their MSA.  In addition, one can
+increase the size of modules as a project proceeds.  Once a project
+has a set of vendors who understand the product, demonstrate open
+source expertise, and have established a track record of delivering
+quality work on time, the benefits of modular contracting decrease
+while the costs remain elevated.  At that point, one might opt for
+larger contract modules, especially as the project grows in scope.
+One might also allocate these larger modules among its existing set of
+vendors as a way to balance risk management with cost management.
+
+
+FURTHER_READING: For a comprehensive view into modular contracting,
+  there is a 2012 guide produced by the Obama administration:
```

Comment:
"the Obama administration" sounds a very US-centric term. Consider alternate wording for non-US audiences.

Reply:
For small wording fixes like this (and the "M&O" one later on), it's very cumbersome to be submitting PRs.  I'd really like an "obvious fix" rule for pushing trivial changes on to `main`.  Thoughts?